Code/Release/SupportFiles/glsl/ByPass130.glsl

```diff
@@ -0,0 +1,13 @@
+#version 130
```

Comment:
This and ByPass.glsl indicate they work with the same version of GLSL but have different implementations. Can we remove one of them?